SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -13,7 +13,7 @@ The table below lists the contents of the Unreal GDK repository after running `S
 | `SpatialGDK/Extras/fastbuild` | Contains files related to [FASTBuild](http://www.fastbuild.org/docs/home.html), an open-source build system that is currently only usable with the GDK by Improbable engineers.
 | `SpatialGDK/Extras/schema` | Contains the [schema files](https://docs.improbable.io/reference/latest/shared/glossary#schema) required for the GDK to interact with SpatialOS.
 | `SpatialGDK/Extras/linting/` | Contains the scripts we use to lint the GDK.
-| `SpatialGDK/Source/SpatialGDK/Public/WorkerSdk/` | Not tracked in git. Contains the [C API worker SDK](https://docs.improbable.io/reference/latest/capi/introduction) headers which are used while building the GDK. You install these when you run `Setup.bat`
-| `SpatialGDK/Source/SpatialGDK/Public` | Contains the public source code of the GDK uplugin.
-| `SpatialGDK/Source/SpatialGDK/Private` | Contains the private source code of the GDK uplugin.
+| `SpatialGDK/Source/Public/WorkerSdk/` | Not tracked in git. Contains the [C API worker SDK](https://docs.improbable.io/reference/latest/capi/introduction) headers which are used while building the GDK. You install these when you run `Setup.bat`
+| `SpatialGDK/Source/Public` | Contains the public source code of the GDK uplugin.
+| `SpatialGDK/Source/Private` | Contains the private source code of the GDK uplugin.
```

Comment:
Would you want to add the directories related to the editor plugin here?

Reply:
I left them out as there shouldn't be too much of interest, but maybe it is worth having both

Reply:
The editor plugin doesn't have additional folders outside a normal editor plugin structure, so I don't think additional explanation is necessary.